Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -112,6 +119,107 @@ void OnGUI()
+                        string nameWOext = r.Replace(asset.Replace(".prefab", ""),""); //remove extension and all previous directories
+                        if (removeVariantText)
+                        {
+                            nameWOext = nameWOext.Replace(" variant", "");
+                        }
+
+                        //Get images 
+                        DirectoryInfo dirInfo = new DirectoryInfo(Application.dataPath + assetBundleFolderLocation + "source");
+                        string thisFileName = Path.GetFileName(asset);
+                        string internalFilePath = "Assets" + assetBundleFolderLocation + dirInfo.Name + "/" + thisFileName;
+                        GameObject prefab = ((GameObject) AssetDatabase.LoadAssetAtPath(internalFilePath, typeof(GameObject)));
+                        //Debug.LogWarning(prefab == null);
+                        //AssetDatabase.ImportAsset(AssetDatabase.GetAssetPath(prefab), ImportAssetOptions.ForceUpdate);
+                        EditorUtility.SetDirty(prefab);
+                        Texture2D prefabPreview = AssetPreview.GetAssetPreview(prefab);
+                        //Debug.LogWarning(prefabPreview == null);
+
+                        // Create a temporary RenderTexture of the same size as the texture
+                        RenderTexture tmp = RenderTexture.GetTemporary(
+                                            prefabPreview.width,
+                                            prefabPreview.height,
+                                            0,
+                                            RenderTextureFormat.Default,
+                                            RenderTextureReadWrite.Default);
+                        // Blit the pixels on texture to the RenderTexture
+                        Graphics.Blit(prefabPreview, tmp);
+                        // Backup the currently set RenderTexture
+                        RenderTexture previous = RenderTexture.active;
+                        // Set the current RenderTexture to the temporary one we created
+                        RenderTexture.active = tmp;
+                        // Create a new readable Texture2D to copy the pixels to it
+                        Texture2D myTexture2D = new Texture2D(prefabPreview.width, prefabPreview.height);
+                        // Copy the pixels from the RenderTexture to the new Texture
+                        myTexture2D.ReadPixels(new Rect(0, 0, tmp.width, tmp.height), 0, 0);
+                        myTexture2D.Apply();
+                        // Reset the active RenderTexture
+                        RenderTexture.active = previous;
+                        // Release the temporary RenderTexture
+                        RenderTexture.ReleaseTemporary(tmp);
+                        byte[] bytes = myTexture2D.EncodeToPNG();
+                        imgPreviews[nameWOext] = bytes;
+                        imgPreviews[nameWOext.ToLower()] = bytes;
```

Comment:
This is the preview image rendering and mapping. Yes it does require the many temp renders due to how the render engine works